app/web/src/components/Ai/GenerateAwsAssetSchemaPanel.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="flex flex-row gap-xs place-items-end">
+    <VormInput
+      ref="cli"
+      v-model="awsCommand.cli"
+      label="CLI"
+      type="text"
+      disabled
+      :maxLength="3"
+      @enterPressed="generateAwsAssetSchema"
+    />
+    <VormInput
+      ref="command"
+      v-model="awsCommand.command"
+      label="Command"
+      type="text"
+      :disabled="generateRequestStatus.isPending"
+      @enterPressed="generateAwsAssetSchema"
+    />
+    <VormInput
+      ref="subcommand"
+      v-model="awsCommand.subcommand"
+      label="Subcommand"
+      type="text"
+      :disabled="generateRequestStatus.isPending"
+      @enterPressed="generateAwsAssetSchema"
+    />
+    <VButton
+      :loading="generateRequestStatus.isPending"
+      loadingIcon="sparkles"
+      loadingText="Generating ..."
+      :requestStatus="generateRequestStatus"
+      tooltip="Generate Schema From AWS Command"
```

Comment:
VButton doesn't have a tooltip prop, so to make this tooltip functional I'll change it to the v-tooltip Vue directive.